software/src/enso/helpers.cpp

```diff
@@ -143,8 +155,20 @@ int set_core_id(std::thread& thread, int core_id) {
 static void print_stats_line(uint64_t recv_bytes, uint64_t nb_batches,
                              uint64_t nb_pkts, uint64_t delta_bytes,
                              uint64_t delta_pkts, uint64_t delta_batches) {
-  std::cout << std::dec << (delta_bytes + delta_pkts * 20) * 8. / 1e6
-            << " Mbps  " << delta_pkts / 1e6 << " Mpps  " << recv_bytes
+  uint64_t rx_tput_mbps = (delta_bytes * 8.) / ONE_MILLION;
+  if(rx_tput_mbps > 0) {
```

Comment:
Why are you handling these two packet sizes in particular?

Reply:
Those are the only two for which I tested (and was able to find the PCAP for in scripts/sample_pcaps). I'll generate for the others as well--128, 256, 512 and 1024--as per figure 9 of the paper. Please let me know if any other sizes need to be supported.